stable/redis/values.yaml

```diff
@@ -40,6 +40,10 @@ args:
 persistence:
   enabled: true
 
+  ## The path the volume will be mounted at, useful when using different
+  ## Redis packages.
```

Comment:
"redis packages" sounds a little confusing to me, I think "image" singular is more clear.

Reply:
You're right, not sure where that came from